measurement/measures/inductance.py

```diff
@@ -0,0 +1,14 @@
+from measurement.base import MeasureBase
+
+__all__ = ["Inductance"]
```

Comment:
It's funny, here you are referencing the correct class name, even though the class does not exist ;)